examples/datasets/plot_opm_rest_data.py

```diff
@@ -0,0 +1,209 @@
+4. Frequency: power spectrum density (Welch), 4 sec window, 50% overlap.
+5. Standardize: normalize by relative power for each source.
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. _bst_omega: https://neuroimage.usc.edu/brainstorm/Tutorials/RestingOmega
+.. _bst_resting: https://neuroimage.usc.edu/brainstorm/Tutorials/Resting
+
+Preprocessing
+-------------
+"""
+# sphinx_gallery_thumbnail_number = 14
+
+# Authors: Denis Engemann <denis.engemann@gmail.com>
+#          Luke Bloy <luke.bloy@gmail.com>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+from mne.filter import next_fast_len
+from mayavi import mlab
+
+import mne
+
+
+print(__doc__)
+
+data_path = mne.datasets.opm.data_path()
+subject = 'OPM_sample'
+
+subjects_dir = op.join(data_path, 'subjects')
+bem_dir = op.join(subjects_dir, subject, 'bem')
+bem_fname = op.join(subjects_dir, subject, 'bem',
+                    subject + '-5120-5120-5120-bem-sol.fif')
+src_fname = op.join(bem_dir, '%s-oct6-src.fif' % subject)
+vv_fname = data_path + '/MEG/SQUID/SQUID_resting_state.fif'
+vv_erm_fname = data_path + '/MEG/SQUID/SQUID_empty_room.fif'
+vv_trans_fname = data_path + '/MEG/SQUID/SQUID-trans.fif'
+opm_fname = data_path + '/MEG/OPM/OPM_resting_state_raw.fif'
+opm_erm_fname = data_path + '/MEG/OPM/OPM_empty_room_raw.fif'
+opm_trans_fname = None
+opm_coil_def_fname = op.join(data_path, 'MEG', 'OPM', 'coil_def.dat')
+
+##############################################################################
+# Load data, resample. We will store the raw objects in dicts with entries
+# "vv" and "opm" to simplify housekeeping and simplify looping later.
+
+raws = dict()
+raw_erms = dict()
+new_sfreq = 90.  # Nyquist frequency (45 Hz) < line noise freq (50 Hz)
+raws['vv'] = mne.io.read_raw_fif(vv_fname, verbose='error')  # ignore naming
+raws['vv'].load_data().resample(new_sfreq)
+raws['vv'].info['bads'] = ['MEG2233', 'MEG1842']
+raw_erms['vv'] = mne.io.read_raw_fif(vv_erm_fname, verbose='error')
+raw_erms['vv'].load_data().resample(new_sfreq)
+raw_erms['vv'].info['bads'] = ['MEG2233', 'MEG1842']
+
+raws['opm'] = mne.io.read_raw_fif(opm_fname)
+raws['opm'].load_data().resample(new_sfreq)
+raw_erms['opm'] = mne.io.read_raw_fif(opm_erm_fname)
+raw_erms['opm'].load_data().resample(new_sfreq)
+# Make sure our assumptions later hold
+assert raws['opm'].info['sfreq'] == raws['vv'].info['sfreq']
+
+##############################################################################
+# Do some minimal artifact rejection just for VectorView data
+
+titles = dict(vv='VectorView', opm='OPM')
+ssp_ecg, _ = mne.preprocessing.compute_proj_ecg(
+    raws['vv'], tmin=-0.1, tmax=0.1, n_grad=1, n_mag=2)
+raws['vv'].add_proj(ssp_ecg, remove_existing=True)
+ssp_ecg_eog, _ = mne.preprocessing.compute_proj_eog(
+    raws['vv'], n_grad=1, n_mag=1, ch_name='MEG0112')
+raws['vv'].add_proj(ssp_ecg_eog, remove_existing=True)
```

Comment:
why ssp_ecg_eog if it it is the output of compute_proj_eog ?

Reply:
Because of the weirdness of how `compute_proj_eog` works -- it returns all projectors, not just the new ones, so in this case it includes the ecg from before and the eog that were newly computed

Reply:
Can we say that in a comment?